### DISCUSSION TITLE
setup the UTF encoding in the pdfplumber open function

### DISCUSSION THREAD

**Discussion by Hammad-Asif** [jsvine/pdfplumber#785] [Q&A]
Hey, I need to extract the text from the pdf. The text is written in Arabic language. But extract text function is returning unknown characters. I have tried pypDF2 for this i need the text with layout so for that pdfplumber is only option for me. kindly help me in this regard. Thanks

**Comment by samkit-jain** [comment]
@Hammad-Asif Appreciate your interest in the library. Could you please share the PDF (redacting any sensitive information) and a reproducible code?
